mods/d2k/maps/atreides-04/map.yaml

```diff
@@ -0,0 +1,415 @@
+MapFormat: 11
+
+RequiresMod: d2k
+
+Title: Atreides 04
+
+Author: Westwood Studios
+
+Tileset: ARRAKIS
+
+MapSize: 68,68
+
+Bounds: 2,2,64,64
+
+Visibility: MissionSelector
+
+Categories: Campaign
+
+LockPreview: True
+
+Players:
+	PlayerReference@Neutral:
+		Name: Neutral
+		OwnsWorld: True
+		NonCombatant: True
+		Faction: Random
+	PlayerReference@Creeps:
+		Name: Creeps
+		NonCombatant: True
+		Faction: Random
+		Enemies: Atreides, Harkonnen, Fremen
+	PlayerReference@Atreides:
+		Name: Atreides
+		Playable: True
+		LockFaction: True
+		Faction: atreides
+		LockColor: True
+		Color: 9191FF
+		Allies: Fremen
+		Enemies: Harkonnen, Creeps
+	PlayerReference@Harkonnen:
+		Name: Harkonnen
+		LockFaction: True
+		Faction: harkonnen
+		LockColor: True
+		Color: FE0000
+		Enemies: Atreides, Creeps, Fremen
+	PlayerReference@Fremen:
+		Name: Fremen
+		LockFaction: True
+		Faction: fremen
+		LockColor: True
+		Color: 9191FF
```

Comment:
Wierd. I remember it being Gray when i tested yesterday.

Whatever it needs to be gray.